tokio-console/args.example

```diff
@@ -6,8 +6,6 @@ ARGS:
             The address of a console-enabled process to connect to.
             
             This may be an IP address and port, or a DNS name.
-            
-            [default: http://127.0.0.1:6669]
```

Comment:
it's a shame that we can no longer show default values in the CLI help. it might be worth manually adding the defaults to the doc comments that generate the help message.

Reply:
I fixed: https://github.com/tokio-rs/console/pull/334/commits/23d396ee7ba1791df1061c2b30126a537e584252